spec/spec_helper.cr

```diff
@@ -25,6 +25,15 @@ macro assert_unauthorized(response)
   fail "expected error response not found" unless found
 end
 
+# Disable all authentication checks by making `DppmRestApi::Actions.has_access?`
+# always return true, then set the authentication back to the default system
+# after yielding to the block.
+def without_authentication!
```

Comment:
Is this ok?

Reply:
Sounds good. Any idea to test the filter in `actions_spec.cr`?
We can fix it in a further PR if we don't find a solutiom yet

Reply:
Well, the access_filter has to take several steps, each of which are already tested more or less. I agree that an integration/regression test on this would be due, but I think it would be better to set that up in a PR dedicated to integration and regression tests once the API is a little more fleshed out.

If you'd prefer, I could set up a test for this method before merging it.

Reply:
I agree. I think we are good to merge, this will be simpler.